cmd/guard.go

```diff
@@ -0,0 +1,83 @@
+package cmd
```

Comment:
1. Rename file to specify what guard it is, e.g. `flutter_config_version_guard`
2. May we hide all these functions from the scope and put them inside a struct, so that we can make a struct in the command (global) and pass method of that struct to the `PreRun`?